test/Concurrency/sendable_methods.swift

```diff
@@ -0,0 +1,186 @@
+// RUN: %target-typecheck-verify-swift -enable-experimental-feature InferSendableMethods
+// RUN: %target-swift-emit-silgen %s -verify -enable-experimental-feature InferSendableMethods -module-name M | %FileCheck %s
```

Comment:
You'd either need to annotate all the declarations as `@available(SwiftStdlib 5.1, *)` or use `-disable-availability-checking` for both RUN commands.